modules/andes-core/broker/src/main/java/org/wso2/andes/amqp/AMQPLocalSubscription.java

```diff
@@ -243,21 +243,22 @@ private void sendMessage(QueueEntry queueEntry) throws AndesException {
             if (amqpSubscription instanceof SubscriptionImpl.AckSubscription) {
 
                 MessageTracer.trace(messageID, "",
-                                    "Sending message " + msgHeaderStringID + " messageID-" + messageID + "-to channel "
-                                    + getChannelID());
+                                    "Sending message " + msgHeaderStringID
```

Comment:
String msgHeaderStringID = (String) queueEntry.getMessageHeader().getHeader("msgID");

msgHeaderStringID is different from message ID. It is used to identify message based on JMS Header. This log makes a correlation between the two.